ssh/tailssh/incubator.go

```diff
@@ -117,6 +122,7 @@ func (ss *sshSession) newIncubatorCommand(logf logger.Logf) (cmd *exec.Cmd, err
 		"--local-user=" + lu.Username,
 		"--remote-user=" + remoteUser,
 		"--remote-ip=" + ci.src.Addr().String(),
+		fmt.Sprintf("--is-se-linux-enforcing=%v", isle),
```

Comment:
rather than muddy up the argv as seen in `ps` with this, and to also get rid of the `isle` variable name that looks like the word isle, how about:

```go
if runtime.GOOS == "linux" && hostinfo.IsSELinuxEnforcing() {
     incubatorArgs = append(incubatorArgs, "--is-se-linux-enforcing")
}
```